### PR TITLE
Adding quad-precision build support

### PR DESCRIPTION
**Pull request by SwayamInSync**
This PR adds quad-precision support to the SLEEF conda-forge package. The changes enable the building and packaging of the quad-precision library (`libsleefquad`) alongside the existing SLEEF libraries.

### CODE REVIEW

**Comment by SwayamInSync** [comment]
PR regarding issue #8

**Comment by SwayamInSync** [comment]
@bluescarni any thoughts on why the windows and Travis CI are failing?

**Comment by bluescarni** [comment]
@SwayamInSync for the PPC64 build failure, I recall reading on the conda-forge gitter channel that there were issues with travis and that the solution was to switch to cross compilation from x86. Pinging the great @h-vetinari to see if they can assist as I my conda-forge-fu is not that strong.

For the Windows build I don't really know...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,17 +20,23 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
     - {{ stdlib("c") }}
     - cmake
     - ninja        # [unix]
+    - make         # [unix]
```
Comment (bluescarni) [inline recipe/meta.yaml]: @SwayamInSync I was wondering why is ``make`` needed here, as the recipe on Unix uses ``ninja`` as generator?

**Comment by h-vetinari** [comment]
> Pinging the great @h-vetinari to see if they can assist as I my conda-forge-fu is not that strong.

Thanks for the flattery, but "stubborn" might be more accurate than "great". 😅 

In any case, PPC is going to run into https://github.com/conda-forge/status/issues/185. Since you're already cross-compiling osx-arm64, the hope would be that you could switch to cross-compilation also for linux-{aarch64,ppc64le}, or in the worst case, emulate them on azure (which is quite a bit slower, but since the build times here are not that big, that would also be an option).

File: recipe/meta.yaml (hunk 1 above)
Comment (SwayamInSync) [reply recipe/meta.yaml]: Thanks for catching that, make sense, I would've just put that there for my local use, will push a refactor commit for this

**Comment by SwayamInSync** [comment]
> > Pinging the great @h-vetinari to see if they can assist as I my conda-forge-fu is not that strong.
> 
> Thanks for the flattery, but "stubborn" might be more accurate than "great". 😅
> 
> In any case, PPC is going to run into [conda-forge/status#185](https://github.com/conda-forge/status/issues/185). Since you're already cross-compiling osx-arm64, the hope would be that you could switch to cross-compilation also for linux-{aarch64,ppc64le}, or in the worst case, emulate them on azure (which is quite a bit slower, but since the build times here are not that big, that would also be an option).

Thanks @h-vetinari , switched to azure provider
```yaml
provider: {linux_aarch64: azure, linux_ppc64le: azure}
```

Travis CI is still failing for `ppc64le`, but is it now to a point where we can merge it?
cc: @bluescarni

**Comment by bluescarni** [comment]
> Travis CI is still failing for `ppc64le`, but is it now to a point where we can merge it? cc: @bluescarni

I think you need to rerender the recipe.

**Comment by SwayamInSync** [comment]
> > Travis CI is still failing for `ppc64le`, but is it now to a point where we can merge it? cc: @bluescarni
> 
> I think you need to rerender the recipe.

Thanks @bluescarni
Pushed the re-rendered one, seems good now

**Comment by bluescarni** [comment]
@SwayamInSync cheers, LGTM, merging.

**Comment by SwayamInSync** [comment]
Ahh I think by mistake I left this for windows 🥲

opening a new PR

**Pull request merged**